slack-notifier.gemspec

```diff
@@ -12,6 +12,7 @@ Gem::Specification.new do |s|
   s.authors       = ["Steven Sloan"]
   s.email         = ["stevenosloan@gmail.com"]
   s.homepage      = "https://github.com/slack-notifier/slack-notifier"
+  s.metadata      = { "changelog_uri" => "https://github.com/slack-notifier/slack-notifier/blob/main/changelog.md" }
```

Comment:
![image](https://github.com/slack-notifier/slack-notifier/assets/1275215/9c8accd2-966a-484a-ac41-03ee75800703)
need to check wether the metadata is supported or not